learningresources/tests/test_api.py

```diff
@@ -61,6 +64,10 @@ def test_create(self):
             create_course(**self.kwargs)
         self.assertTrue(course_count() == after)
 
+        # NOT NULL constraint fails on org
+        with self.assertRaises(IntegrityError):
```

Comment:
When I use tox it uses sqlite3. I'll see if I can add a check for "NULL" in the string